src/main/java/com/taskadapter/redmineapi/bean/TimeEntry.java

```diff
@@ -31,6 +34,7 @@
     public final static Property<Date> SPENT_ON = new Property<>(Date.class, "spentOn");
     public final static Property<Date> CREATED_ON = new Property<>(Date.class, "createdOn");
     public final static Property<Date> UPDATED_ON = new Property<>(Date.class, "updatedOn");
+	public static final Property<List> CUSTOM_FIELDS = new Property<>(List.class, "custom_fields");
```

Comment:
how about using Set instead of List? do you expect duplicate entries there? if yes, then how would they be resolved?